src/app.py

```diff
@@ -81,13 +83,16 @@ def store_data(json_data, csv_data):
 		raise
 
 
-def source_urls_to_pdfs(source_urls):
-	logging.info("Converting source websites into PDFs")
+def source_urls_to_pdfs(source_urls, folder, names=None):
```

Comment:
Since this function is now no longer specific to source urls, I would rename it to something like `urls_to_pdfs`